Gemfile

```diff
@@ -4,3 +4,4 @@ gem 'rake'
 gem 'minitest', '~> 5.0'
 gem 'nokogiri', '~> 1.8'
 gem 'activesupport'
+gem 'pry'
```

Comment:
This is really a development dependency. Might be a small optimization, but do we need to depend on it? Can we skip it in CI?

Reply:
Can remove in the final version, I just do a lot of debugging.

Reply:
Might make sense to introduce a `development` group and do `bundle install --without development` in CI.